docs/category.yaml

```
@@ -226,3 +226,70 @@ paths:
+        required: false
+        schema:
+          type: integer
+          default: 20
+          minimum: 1
+        description: Number of categories per page.
+    responses:
+      '200':
+        description: A paginated list of category names
+        content:
+          application/json:
+              schema:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)